src/langcheck/metrics/custom_text_quality.py

```diff
@@ -22,6 +25,9 @@ def custom_evaluator(
     score_map: dict[str, float],
     template_path: str,
     language: str,
+    *,
+    additional_params: dict[str, IndividualInputType] | None = None,
+    additional_params_to_prompt_var_mapping: dict[str, str] | None = None,
```

Comment:
And "additional_input_name_to_prompt_var_mapping" here

Reply:
Also updated